### ISSUE TITLE
Can't get rid of double space before and parentheses around citation

### ISSUE THREAD

**Issue by lazzalazza** [jgm/pandoc#6871]
I'm converting from org-mode to docx via pandoc and I can't figure out how to avoid this problem.
The following org-ref citation:

bla bla bla cite:author1999a.

gives me the following result when exporting using pandoc --citeproc:

bla bla bla [double space] (Author, Title, etc.).

I would like to avoid both the double space and the parentheses around the citation.
Btw, I use a csl style which is perfectly handled by citeproc, the only problem is that I can't remove the double spaces and the parentheses.

**Comment by jgm** [comment]
This is more of a pandoc issue, if it's an issue at all, so I'm moving this there.
Can you provide exact instructions to reproduce?  I just tried and don't see a double space.  There isn't a way to remove the parens; they are called for by the style.

**Comment by lazzalazza** [comment]
Sorry, you're perfectly right: I simply panicked because pandoc-citeproc treated the csl style I was using exactly as an "author-date" class (as I found out later, it was a "note"). Therefore the "problem" is that the new citeproc library is really interpreting a feature that the old tool simply ignored. Thank you so much for your help: this issue is not an issue and therefore it can be closed... And, once again, sorry for disturbing you out of panic.
